src/tools/storage-key-recipe-resolver.ts

```diff
@@ -38,7 +38,7 @@ export class StorageKeyRecipeResolver {
    * Produces resolved recipes with storage keys.
    *
    * @throws Error if recipe fails to resolve on first or second pass.
-   * @yields Resolved recipes with storage keys
+   * @returns Resolved Recipe (with Storage Keys).
```

Comment:
the return value is recipe array, please, update comment to plural

Reply:
I think I just missed typing an `s`. Thanks for catching this.